data/daniel-jp.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "Daniel J. Paulino",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Software and Web Dev | Data",
+    "avatar": "https://github.com/daniel-jp.png",
+    "links": [{
+            "name": "Connect me on Linkedin",
+            "url": "https://www.linkedin.com/in/daniel-j-paulino-737808198/",
+            "icon": "FaLinkedin"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/Daniel28725148",
+            "icon": "FaTwitter"
+        },
+
+        {
+            "name": "Follow me on Instagram",
+            "url": "https://www.instagram.com/daniel_j_paulino/",
+            "icon": "FaInstagramSquare"
+        },
+
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/SaraJaoude",
```

Comment:
Hi @daniel-jp Kindly remove this. This is Sara's twitter account. It is not yours.